test/test_kramdown.rb

```diff
@@ -4,10 +4,24 @@
 require "rouge"
 
 class TestKramdown < JekyllUnitTest
+  def fixture_converter(config)
+    site = fixture_site(
+      Utils.deep_merge_hashes(
+        {
+          "markdown" => "kramdown"
```

Comment:
```suggestion
          "markdown" => "kramdown",
```

Reply:
I had already pushed the change. But GitHub never picked it up due to the partial outage..